### PR TITLE
[bitnami/harbor] Sync with official Harbor chart

### PR DESCRIPTION
**Pull request by marcosbc**
**Description of the change**

Sync with official Harbor helm chart

**Benefits**

Support latest features and fixes

**Possible drawbacks**

None

**Applicable issues**

None

**Additional information**

**Checklist** [Place an '[X]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

**Comment by alemorcuq** [comment]
Since this is adding some functionality, it shouldn't be a patch version.

**Comment by carrodher** [comment]
I just resolved some conflicts (major bump because of https://github.com/bitnami/charts/pull/1290) and updated the minor version

File: bitnami/harbor/values.yaml [hunk 1]
```diff
@@ -311,6 +314,8 @@ service:
       # The service port Notary listens on. Only needed when notary.enabled
       # is set to true
       notaryPort: 4443
+    annotations: {}
```
Comment (carrodher) [inline bitnami/harbor/values.yaml]: The `loadBalancer` section in the _values-production.yaml_ doesn't contains those values

File: bitnami/harbor/values.yaml [hunk 2]
```diff
@@ -327,6 +332,14 @@ service:
 # If Harbor is deployed behind the proxy, set it as the URL of proxy
 externalURL: https://core.harbor.domain
 
+## UAA Authentication Options
+# If you're using UAA for authentication behind a self-signed
+# certificate you will need to provide the CA Cert.
+# Set uaaSecretName below to provide a pre-created secret that
+# contains a base64 encoded CA Certificate named `ca.crt`.
+# uaaSecretName:
+
```
Comment (carrodher) [inline bitnami/harbor/values.yaml]: Unnecessary line in both files

File: bitnami/harbor/values.yaml (hunk 1 above)
Comment (marcosbc) [reply bitnami/harbor/values.yaml]: Good point, it seems I added them to the wrong location.

File: bitnami/harbor/values.yaml (hunk 2 above)
Comment (marcosbc) [reply bitnami/harbor/values.yaml]: Removed

**Pull request merged**